webpack.config.js

```diff
@@ -242,6 +242,7 @@ module.exports = {
 		symlinks: false,
 		alias: {
 			'react-dom': '@hot-loader/react-dom',
+			'wcs-client': path.resolve( __dirname, 'client' ),
```

Comment:
I'm hoping this prefixed alias will help identify where the imports are coming from

Reply:
Do you need to do this for the [sass-loader](https://github.com/Automattic/woocommerce-services/blob/378862c4df16230c89ae2cbbe9ad7423cd667da0/webpack.config.js#L33-L41) as well?

Reply:
Sorry, I'm not sure I'm following - can you expand?

If it helps, the Webpack resolver also parses `.scss` files. I.e.: no additional changes are needed if you want to use `@import '~wcs-client/components/some/path/to/file.scss'` in a `.scss` file